src/orderly/run.py

```diff
@@ -56,6 +62,36 @@ def _validate_src_directory(name, root):
     return path
 
 
+def _validate_parameters(given, defaults):
+    if given is None:
+        given = {}
+
+    if given and not defaults:
```

Comment:
Do we need this check? Isn't that just a special case of the unknown parameters?

Reply:
it's a better error message though IMO